### DISCUSSION TITLE
Borg Backup failing regularly after "crash"

### DISCUSSION THREAD

**Discussion by nfrawley93** [nextcloud/all-in-one#1461] [Questions] [answered]
### Steps to reproduce
1. Run any form of backup (reoccurring or manual) through AIO
2. Check logs after failure indication to see this
3. ![Screenshot 2022-11-29 221553](https://user-images.githubusercontent.com/10273064/204706230-fe95fc22-b65e-4d45-971b-f2779090ce84.jpeg)

### Expected behavior 
Complete daily backup

### Actual behavior 
Long story short I upgraded to new hardware recently. Including a large main HDD. Thought this would be a good time to test a full borg restore. It was super easy and worked great. No issues what-so-ever.
A day or so later while running a backup the server was interrupted (power issue) and when it started up again borg started to fail to backup. After trying and failing to troubleshoot. One of which method just had me deleting the lock file which worked for a single backup then started failing again.
I got a different HDD and mounted to the same spot and completely started borg backup over resetting everything. This seemed to work. Even got a few days of recurring backups out of it. I haven't had any issues up until yesterday where I got the notification that it failed. No power issues or disconnects or interrupts that I am aware of. Not only that but also it wont show the past few days of backups in the drop down anymore.
![Screenshot 2022-11-29 222222](https://user-images.githubusercontent.com/10273064/204706889-7db1b20f-551b-4fc8-92dc-dec6347e21a5.jpeg)

Before all of this I had no issues with backups for months, and even after the change (resetting to scratch) I had a week or so of perfect backups without fail.

### Host OS 
Nextcloud-AIO Virtual Machine

#### Nextcloud AIO version 
24.0.7,
Nextcloud AIO 3.0.0

#### Current channel 
Nextcloud AIO 3.0.0

**Comment by szaimen** [comment, marked as answer]
Yeah, power failures are the worst.

You probably should be able to fix it with:

```
# Install borgbackup on the host
sudo apt update && sudo apt install borgbackup

# break the lock (if you are using the default backup location /mnt/backup/borg)
sudo borg break-lock /mnt/backup/borg
```

See https://borgbackup.readthedocs.io/en/stable/usage/lock.html#borg-break-lock

**Reply by nfrawley93** [reply]
If it happens again I will post the exact wording, but that got me to the next step where I got another issue about it being unable to access some /.cache/.... file. I went ahead and moved my "failing" borgbackup off device and then went in and change/deleted config file for borg to essentially reset it (following from this [d01eed9](https://github.com/nextcloud/all-in-one/commit/d01eed902ad31f71470d3211d1c8674b78334981) commit I believe). It created a whole new backup from scratch (nearly a TB so waiting on that was a bit nerve racking) It has been working well for the past 2 weeks since this happened.

I am worried that I will have this same issue if there is some sort of outage during a future backup but it seems that the backup itself was fine.
